client/src/apps/code-collabo/helpers/appInfo.ts

```diff
@@ -1,15 +1,28 @@
 import { capitalizeAllFirstLetters, stringToUrlStart } from '@/apps/shared/helpers/transform';
 
+interface Page {
+  name: string;
+  route: string;
+}
+
+interface AppInfo {
+  id: number;
+  name: string;
+  pages: {
+    [key: number]: Page;
+  };
+}
+
```

Comment:
**_Review comment 2_**

These interfaces appear to be present in 3 different files @Akpjunior94 
Better to be inside one file and imported everywhere. The file you create for the interface should be in the shared folder's helper folder.

Reply:
Noted will update that as well

Reply:
Done @Ifycode please check.